### DISCUSSION TITLE
Tag-list based events

### DISCUSSION THREAD

**Discussion by wattzor** [frangoteam/FUXA#1145] [Ideas]
Hello!, Just want to say that your project looks amazing. In the past i have used https://kiona.com/products/web-port alot and its very simluare to your project. Some key feature that you could take not off:

Tag-based events.

For example you drag out an object and call it "TAGG_XXX_HVAC"

Then webport will search the available tags,

and based what it find with that name for example:

TAGG_XXX_HVAC_AL
TAGG_XXX_HVAC_V

and so on.

This will automatically be added to the object. And some predefined object will behave in a way.

For example _AL = 1 this will say that the object is in ALARM mode and trigger it to go red for example
_V may be the indication that the pump is running and it will turn it green.

Whit this predefined things production is very fast and you can copy past project fast. Usually you just change the tagg name of the object.

**Comment by unocelli** [comment]
Hi, I suppose that is the same feature #1158

**Reply by wattzor** [reply]
Kind of the same idea yes. Would save alot of time,

And or the poisbility to create objects direct from a tag-list

So basicly 90% of your work is done in a CSV (Much faster to edit and change)
